src/rofi_rbw/rofi_rbw.py

```diff
@@ -104,9 +104,11 @@ def __parse_arguments(self) -> argparse.Namespace:
                     "Alt+2:type:username",
                     "Alt+3:type:password",
                     "Alt+4:type:totp",
+                    "Alt+5:type:notes",
                     "Alt+c:copy:password",
                     "Alt+u:copy:username",
                     "Alt+t:copy:totp",
+                    "Alt+n:copy:notes",
```

Comment:
I'm not sure I'd add this shortcut by default. The info line is already long, so I'd rather leave it out of the default configuration. Interested users can still add it manually.

What do you think?